frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -235,14 +235,14 @@ const Input = (props: TaipyInputProps) => {
                       endAdornment: (
                           <div style={verticalDivStyle}>
                               <IconButton
-                                  data-testid="stepper-up-spinner"
+                                  aria-label={"Increment " + id}
```

Comment:
Aria-label is used by automatic readers to help people with disabilities. 
We need to give it a value that is meaningful
I propose `increment value` (no id)

Reply:
updated